README.md

```diff
@@ -40,6 +38,15 @@ CMD ["/opt/jboss/wildfly/bin/standalone.sh", "-b", "0.0.0.0", "-bmanagement", "0
 1. `docker build -t your-image-tag .`
 1. `docker run --link wildfly:wildfly -d your-image-tag`
 
+## Building the docker image
+
+From in the `didor/` dir
+
+```
+> docker build -t quay.io/democracyworks/didor .
+> docker push quay.io/democracyworks/didor
```

Comment:
This is how we do this internally, but not how someone using this open source code would do it. Can we make these instructions more generic? Something like the following should be sufficient:

```
docker build -t didor .
```
Or you can pull it from `quay.io/democracyworks/didor`